source/Octopus.Manager.Tentacle/TentacleConfiguration/SetupWizard/SetupTentacleWizardModel.cs

```diff
@@ -777,7 +777,7 @@ void UpdateTenants()
                 AreTenantsSupported = spaceSpecificData.AreTenantsSupported;
                 PotentialTenantTags = spaceSpecificData.TenantTags.SelectMany(tt => tt.Tags.Select(t => t.CanonicalTagName)).ToArray();
                 UpdateSelection(SelectedTenantTags, PotentialTenantTags);
-                PotentialTenants = spaceSpecificData.Tenants.Select(tt => tt.Name).ToArray();
+                PotentialTenants = spaceSpecificData.Tenants.Select(tt => tt.IsDisabled ? $"{tt.Name} (disabled)" : tt.Name).ToArray();
```

Comment:
When you select a disabled tenant does it actually generate the correct value in the script output?

Reply:
The proper way to do this would be to change all the usages of `PotentialTenants` over to a class that holds the name and disabled flag in different properties, the selected value is still just the name, but the view can render the disabled property seperately

Reply:
@benPearce1 First suggestion aligns with my thinking, too. Plan is to go with the first approach and lift the "display name" logic into a method on the new class.

Reply:
@benPearce1 Tested new approach and the script has the correct value now 👍 .